teuthology/task/install.py

```diff
@@ -702,28 +702,45 @@ def rh_install(ctx, config):
     :param ctx: the argparse.Namespace object
     :param config: the config dict
     """
+    yaml_path = None
+    # Look for rh specific packages in <suite_path>/rh/downstream.yaml
```

Comment:
I think the format of downstream.yaml should be documented. Perhaps in the docstring of this function, but there might be a better place.